.github/workflows/build.yml

```
@@ -38,4 +50,4 @@ jobs:
             --with-ossp-uuid  --with-libxml --with-libxslt  --with-perl \
             --with-icu --with-libnuma --enable-injection-points --enable-nls
     - name: compile
-      run: make COPT="-Wshadow=compatible-local -Wall -Werror -Wno-stringop-truncation"
+      run: make -j$(nproc) COPT="-Wshadow=compatible-local -Wall -Werror -Wno-stringop-truncation"
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2046:warning:1:8: Quote this to prevent word splitting

(shellcheck)